src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -51,5 +51,15 @@ public sealed class ZipkinExporterOptions
         /// </summary>
         public int? MaxPayloadSizeInBytes { get; set; } = DefaultMaxPayloadSizeInBytes;
 #endif
+
+        /// <summary>
+        /// Gets or sets the exporter type for Zipkin Exporter.
+        /// </summary>
+        public ExporterType ExporterType { get; set; } = ExporterType.BatchExportProcessor;
```

Comment:
I think this name is a bit confusing because if you asked someone what type of exporter they were using they would probably say Zipkin or Jaeger, etc. How about `ProcessorType` with `Simple` and `Batch` definitions?

Reply:
Like it. Maybe even `ExportProcessorType` to be even more explicit.

Reply:
Fixed this.